project_task_stage_allow_timesheet/__manifest__.py

```diff
@@ -0,0 +1,22 @@
+# Copyright 2018 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Project Task Stage Allow Timesheet',
+    'summary': """
+        Allows to tell that a task stage is opened for timesheets.""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Odoo Community Association (OCA), ACSONE SA/NV',
+    'website': 'https://github.com/OCA/hr-timesheet',
+    'depends': [
+        'hr_timesheet',
+        'project',
+    ],
+    'data': [
+        'views/account_analytic_line.xml',
+        'views/project_task_type.xml',
+    ],
+    'demo': [
```

Comment:
@benwillig Please remove empty demo block

Reply:
What's the problem with empty blocks? They aren't displayed anywhere.

Reply:
Just formating, no need to add them since they are empty